docs/en/operation/Journal.md

```diff
@@ -300,6 +300,23 @@ $ ./bin/alluxio fsadmin journal quorum remove -domain <MASTER | JOB_MASTER> -add
 
 3- Verify that the removed member is no longer shown in the quorum info.
 
+##### Transferring leadership to a specific master
+To aid in debugging and to add flexibility, it is possible to manually change the leader of an embedded journal cluster.
+
+1- Check current quorum state:
+```console
+$ ./bin/alluxio fsadmin journal quorum info -domain MASTER
+```
+This will print out node status for all currently participating members of the embedded journal cluster. You should select one 
+of the `AVAILABLE` masters.
+
+2- Transfer the leadership to an available master:
```

Comment:
if you keep the numbered sections together (no empty new lines in between), you dont need to manually increment them. ex:
```
1. first thing
1. second thing
1. third thing
```
renders as
```
1. first thing
2. second thing
3. third thing
```

Reply:
It seems I cannot take advantage of this feature because of the code blocks found in between items in the numbered list